sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.13.0-beta.1 (Unreleased)
 
 ### Features Added
+- Added `AzurePipelinesCredential` to support [Microsoft Entra Workload ID](https://learn.microsoft.com/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml).
```

Comment:
You'll want to add something like "in Azure Pipelines service connections" at the end of this sentence.